code/modules/library/lib_machines.dm

```diff
@@ -776,8 +776,13 @@
 	icon_state = "binder"
 	desc = "Only intended for binding paper products."
 	density = TRUE
+
+	/// Are we currently binding a book?
 	var/busy = FALSE
 
+	/// Name of the author for the book, set by scanning your ID.
+	var/scanned_id
```

Comment:
Can we make this `scanned_name` or something similar? This implies to me it's an ID object